pegasus/migrations/123_add_grade_displayweights_to_tutorials.rb

```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
+  change do
+    alter_table(:tutorials) do
+      add_column :displayweight_k5, String
+      add_column :displayweight_middle, String
+      add_column :displayweight_high, String
```

Comment:
Should these be `Integer` to match the existing displayweight column?

```
mysql> describe tutorials;
+-------------------------------------+---------------+------+-----+---------+----------------+
| Field                               | Type          | Null | Key | Default | Extra          |
+-------------------------------------+---------------+------+-----+---------+----------------+
| id                                  | int(11)       | NO   | PRI | NULL    | auto_increment |
| code                                | varchar(255)  | NO   | UNI | NULL    |                |
| short_code                          | varchar(255)  | NO   | UNI | NULL    |                |
| displayweight                       | int(11)       | NO   |     | NULL    |                |
```

Reply:
Oops, yes.  Fixed.